packages/protocol/CHEATSHEET.md

```diff
@@ -0,0 +1,47 @@
+# Cheatsheet
+
+Covers changes in `package.json`
```

Comment:
Maybe add a bit more context? E.g.

Covers changes in `package.json` scripts, introduced starting with PR #11369. This is a temporary document, slated to be removed after the migration away from Truffle is completed, when `package.json` is expected to be further simplified.

^ Assuming the above is true